app/lib/notifications.server.ts

```diff
@@ -34,12 +34,16 @@ const sendNotification = async (emailAddress: string, subject: string, text: str
   try {
     const transport = initializeTransport();
     logger.debug(`Sending notification to ${emailAddress}`);
-    return await transport.sendMail({
-      from: NOTIFICATIONS_EMAIL_USER,
-      to: emailAddress,
-      subject,
-      text,
-    });
+    return await transport
+      .sendMail({
+        from: NOTIFICATIONS_EMAIL_USER,
+        to: emailAddress,
+        subject,
+        text,
+      })
+      .catch((error) => {
```

Comment:
Would the catch on line 47 not catch this?

Reply:
If I remove the `try/catch` block and let's say the `NOTIFICATIONS_EMAIL_USER` is not defined, it throws an error instead of a warning. Could you test it out and let me know?

Reply:
I am unsure if we should throw an error.

Reply:
I think you can remove the await from line 37 and simply return the promise as @humphd suggested above